### ISSUE TITLE
Flickering of transition Window Animations

### ISSUE THREAD

**Issue by shaktishekhar** [CyanogenMod/android_device_oneplus_oneplus2#16]
@Grarak

Flickering of transition Window Animations while opening apps like whatsapp,playstore,Twitter,Bluemail etc..
i can confirm that this issue is only present in cm 13 official nightly .

Thank You

Regards'

Shakti Shekhar

@shaktishekhar "xda" id

Logcat link = [goo.gl/RzYjxp](url)
Sample Video of issue in link = [goo.gl/McN22H](url)
"Copy The Link and paste it in Address bar don't click"

**Comment by yash53** [comment]
Yes was facing this too, earlier on unofficial build now on official build too.

**Comment by DerkSoul** [comment]
Yes i want a perfect rom. Not this half hazard type roms. This flickering issue has been continuing since months and no developer is giving an eye over it. Please do this as soon as possible. I agree with shakti shekhar as above commented. Hope you solve as soon as possible. Thank You

**Comment by abhishekgarg294** [comment]
i also facing this issue......please resolve it as soon as possible...u guys are superstar for our life....making our life easy by making awsome rom.....please solve this issue.....thnk you so much......

**Comment by shaktishekhar** [comment]
@Grarak nice closed all issues blocked also without resolving them nice :)

**Comment by Grarak** [comment]
@shaktishekhar You are welcome

**Comment by shaktishekhar** [comment]
Lol.. Your Welcome..
